postmortems/000 - Postmortem Template.md

```diff
@@ -0,0 +1,37 @@
+# 000 - Postmortem for [descriptive name here]
+
+## Level: [Internal, Outage, Corruption]
+
+## Author(s)
+
+- Some Body <oncetoldme@iasql.com>
+
+## Summary
+
+This is a postmortem template for outages. In this section you should put a short summary of what happened and how it was resolved. Only one or two paragraphs at most.
+
+Also, to explain the levels. An "Internal" outage is when our internal dev processes are broken and negatively impact the team. An "Outage" level is when service is interrupted for users, but zero impact on customer cloud accounts occurred. A "Corruption" level we hope to never see, which would imply accidental destruction of cloud resources in a customer account caused by us.
+
+## Timeline
+
+- **2022-09-14**: A timeline of events that occurred. No need to be super precise, but please use the ISO date format and UTC time if necessary since we're all over the globe.
+
+## Detection
+
+Describe in a paragraph how the issue was found out.
+
+## Response
+
+Describe in a paragraph or two what was done to resolve the issue.
+
+## Cause
+
+Try to describe a root cause for the issue in a paragraph.
+
+## Remedy
```

Comment:
Oh, this may be terminology confusion. As a "postmortem" I considered the initial cure to end the outage would be in the "timeline" and "response" sections, and this "remedy" would be about the *suggested* steps to take as a response to the postmortem.

I can rename "remedy" to "prevention" if you think that's clearer.

Reply:
Yes, I think that would be more clear. At least for me!